app/Http/Controllers/HomeController.php

```diff
@@ -96,11 +96,22 @@ public function quickSearch()
         $result = [];
 
         if ($quickSearch->hasQuery()) {
+            $error = null;
             foreach ($searches as $mode => $search) {
                 if ($search === null) {
                     continue;
                 }
-                $result[$mode]['total'] = $search->count();
+
+                if ($error === null && $search->getError() !== null) {
+                    $error = $search->getError();
+                }
+
+                if ($error !== null) {
+                    $search->fail($error);
+                    $result[$mode]['total'] = 0;
+                } else {
+                    $result[$mode]['total'] = $search->count();
+                }
```

Comment:
considering failed response already has 0 count...
```suggestion
                }
                
                $result[$mode]['total'] = $search->count();
```